website/blog/2020-11-10-lithium.md

```diff
@@ -147,10 +147,19 @@ Both new options are disabled by default, since they add a great number of
 information, which might not be necessary for all users. Full name with package
 for each of the additional types or values will be available on hover.
 
+These options can be set in Metals options in VS Code or in Emacs
+([lsp-metals])(https://github.com/emacs-lsp/lsp-metals) using:
+
+```
+(customize-set-variable 'lsp-metals-show-implicit-arguments t)
```

Comment:
I'd probably just use `setq` like:

```elisp
(setq lsp-metals-show-implicit-arguments t)
```

Or we can avoid it at all, because emacs has its own UI to modify custom user variables that can be shown with `customize-variable`.

Reply:
Changed to just mentioned variable names. Is this ok? I am totally lost when it comes to emacs :sweat: